### PR TITLE
Add win-x64 support

### LINKED ISSUE #138: Use 64-bit sqlite3.dll for win-x64
Currently we only support x86 on Windows.

### PR DESCRIPTION
**Pull request by natemcmaster**
Fix #138

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -15,18 +15,26 @@ k-standard-goals
...
-            Tuple.Create("sqlite3.dll", sqliteDllPath),
+            Tuple.Create("sqlite3.dll", "src/Microsoft.Data.Sqlite/runtimes/win-x86/native/sqlite3.dll"),
             Tuple.Create("sqlite3.dll", "src/Microsoft.Data.Sqlite/runtimes/win/native/x86/sqlite3.dll"));
         DownloadAndExtract(
             "https://sqlite.org/2015/sqlite-uap-3081101.vsix",
             Tuple.Create("Redist/Retail/x86/sqlite3.dll", "src/Microsoft.Data.Sqlite/runtimes/win10-x86/native/sqlite3.dll"),
             Tuple.Create("Redist/Retail/x64/sqlite3.dll", "src/Microsoft.Data.Sqlite/runtimes/win10-x64/native/sqlite3.dll"),
             Tuple.Create("Redist/Retail/ARM/sqlite3.dll", "src/Microsoft.Data.Sqlite/runtimes/win10-arm/native/sqlite3.dll"));
+        DownloadAndExtract(
+            "http://system.data.sqlite.org/blobs/1.0.98.0/sqlite-netFx451-static-binary-x64-2013-1.0.98.0.zip",
```
Comment (bricelam) [inline makefile.shade]: See [my comments](https://github.com/aspnet/Microsoft.Data.Sqlite/issues/138#issuecomment-140780061) on #138

File: makefile.shade [hunk 2]
```diff
@@ -15,18 +15,26 @@ k-standard-goals
         K("test -parallel none", "test/Microsoft.Data.Sqlite.Tests", "default -r CoreCLR");
     }
 
-var sqliteDllPath='src/Microsoft.Data.Sqlite/runtimes/win-x86/native/sqlite3.dll'
-#sqlite-download target='initialize' if='!File.Exists(sqliteDllPath)'
+#test-x64 target='xunit-test' if='!IsMono'
+    @{
+        K("test -parallel none", "test/Microsoft.Data.Sqlite.Tests", "default -r clr -arch x64");
+        K("test -parallel none", "test/Microsoft.Data.Sqlite.Tests", "default -r coreclr -arch x64");
```
Comment (bricelam) [inline makefile.shade]: Why disable parallelization?

File: makefile.shade [hunk 3]
```diff
@@ -15,18 +15,26 @@ k-standard-goals
...
+            Tuple.Create("sqlite3.dll", "src/Microsoft.Data.Sqlite/runtimes/win-x86/native/sqlite3.dll"),
             Tuple.Create("sqlite3.dll", "src/Microsoft.Data.Sqlite/runtimes/win/native/x86/sqlite3.dll"));
         DownloadAndExtract(
             "https://sqlite.org/2015/sqlite-uap-3081101.vsix",
             Tuple.Create("Redist/Retail/x86/sqlite3.dll", "src/Microsoft.Data.Sqlite/runtimes/win10-x86/native/sqlite3.dll"),
             Tuple.Create("Redist/Retail/x64/sqlite3.dll", "src/Microsoft.Data.Sqlite/runtimes/win10-x64/native/sqlite3.dll"),
             Tuple.Create("Redist/Retail/ARM/sqlite3.dll", "src/Microsoft.Data.Sqlite/runtimes/win10-arm/native/sqlite3.dll"));
+        DownloadAndExtract(
+            "http://system.data.sqlite.org/blobs/1.0.98.0/sqlite-netFx451-static-binary-x64-2013-1.0.98.0.zip",
+            Tuple.Create("SQLite.Interop.dll", "src/Microsoft.Data.Sqlite/runtimes/win/native/x64/sqlite3.dll"));
```
Comment (bricelam) [inline makefile.shade]: Should also go to `runtimes\win-x64\native\sqlite3.dll`

File: makefile.shade (hunk 2 above)
Comment (natemcmaster) [inline makefile.shade]: CoreCLR on linux.

**Comment by natemcmaster** [comment]
See #138

**Pull request closed without merging**